packages/orchestrator-ui-components/src/components/WFOSubscriptionsList/WFOSubscriptionsList.tsx

```diff
@@ -16,7 +16,7 @@ import {
     WFOTableControlColumnConfig,
     WFOTableWithFilter,
 } from '../WFOTable';
-import { SubscriptionListItem } from './types';
+import { SubscriptionListItem } from '../../types';
```

Comment:
I don't think it is a good idea to put all types into the same file. That file is getting large. The reason I place the SubscriptionListItem close to the component is that it is only used here. More specific, a Subscription as we know in the backend should (my opinion) be placed in the types folder, but a so-called view-type that only exists to assist rendering the table should be placed here

Reply:
Placing them in one file is because I was getting circular dependencies as mentioned in another comment. 

On the other part of your argument. I agree with your point on trivial things linke componentProps but a ListItem is general enough to put with the other types imo. If you are looking for types when casually scanning the  package there should be one place to find them